src/components/Table/Table.vue

```diff
@@ -9,7 +9,11 @@
                                      :sticky-header="stickyHeader"
                                      :sorted="sortedBy === column.id"
                                      :sorted-desc="sortedDesc"
-                                     @sorted="onSorted(column)"/>
+                                     @sorted="onSorted(column)">
+                        <template v-slot:[headerSlotName(column.id)]>
```

Comment:
I don't think you need to use named slot inside `TableHeaderCell` (or at least not dynamic name), because it's actually a separate component for every cell. In other words `TableHeaderCell` has only one slot anyway, only `Table` needs those named `header-xxx` slots.